src/python/pants/core/goals/export.py

```diff
@@ -88,35 +93,34 @@ class Export(Goal):
 async def export(
     console: Console,
     targets: Targets,
-    export_subsystem: ExportSubsystem,
     workspace: Workspace,
     union_membership: UnionMembership,
     build_root: BuildRoot,
     dist_dir: DistDir,
 ) -> Export:
-    request_types = cast(
-        "Iterable[type[ExportableDataRequest]]", union_membership.get(ExportableDataRequest)
-    )
+    request_types = cast("Iterable[type[ExportRequest]]", union_membership.get(ExportRequest))
     requests = tuple(request_type(targets) for request_type in request_types)
-    exportables = await MultiGet(
-        Get(ExportableData, ExportableDataRequest, request) for request in requests
-    )
+    all_results = await MultiGet(Get(ExportResults, ExportRequest, request) for request in requests)
```

Comment:
Same comment re renaming throughout, it just makes the salient changes harder to review.